Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            String name;
+            int age;
+            double height;
+            double weight;
```

Comment:
Declaring variables at this point in code forces a programmer to memorize the type until it is actually used. I'd heavily recommend to declare variables as late as possible so you (and other people) do not struggle reading your code.
Also mind the difference between String and string! :)